Animations/README.md

```diff
@@ -0,0 +1,124 @@
+# Animations
+
+[RecyclerViewExtensions](https://github.com/Doist/RecyclerViewExtensions) module providing utilities for animations.
+
+## `AnimatedAdapter`
+
+`Adapter` implementation that provides an `animateDataSetChanges()` method to animate between two data sets. It uses `DataSetDiffer`.
+
+To animate data set changes, `AnimatedAdapter` requires the following method implementations:
+* `getItemId(int)`, providing unique ids for each item (see:  [`RecyclerView.Adapter#hasStableIds()`](https://developer.android.com/reference/androidx/recyclerview/widget/RecyclerView.Adapter.html#hasStableIds())).
+* `getItemContentHash(int)`, providing a hash of the item properties bound in the adapter, that changes when the properties change.
```

Comment:
It could be useful to (re-)emphasise that all UI-affecting state needs to be involved in the hash computation. E.g. appending something like "All mutable state that affects the views when binding should be part of the hash input."

Reply:
Love this! Will add.